src/pages/index.module.css

```diff
@@ -1,18 +1,145 @@
+
+
 /**
  * CSS files with the .module.css suffix will be treated as CSS modules
  * and scoped locally.
  */
- .cardsWrapper {
+
+ .cards {
+    margin: 0 auto;
+    display: grid;
+    gap: 2.4rem;
+}
+
+.actionCards {
+    margin: 0 auto;
+    display: grid;
+    gap: 2.4rem;
+    width: 100%;
+}
+
+.sectionLayoutWrapper {
     display: flex;
-    align-items: stretch;
-    flex-wrap: wrap;
-    box-sizing: border-box;
+    flex-direction: column;
+    align-items: flex-start;
+    align-self: stretch;
+    gap: 2.4rem;
+}
+
+.bannerContentActions {
```

Comment:
Just a nit the titles in these cards are slightly misaligned 
![image](https://github.com/apify/apify-docs/assets/77627332/77ca74f6-761f-4767-8272-8ae16b25091a)